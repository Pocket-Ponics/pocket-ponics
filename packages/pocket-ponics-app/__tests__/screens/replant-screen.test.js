import React from 'react'
import { shallow } from 'enzyme'

import ReplantScreen from '../../screens/replant-screen'

describe('Replant Screen', () => {
	test('renders correctly', () => {
		global.plants = []
		const tree = shallow(
			<ReplantScreen navigation={{ getParam: jest.fn() }}/>
		).html()
		expect(tree).toMatchSnapshot()
	})
})