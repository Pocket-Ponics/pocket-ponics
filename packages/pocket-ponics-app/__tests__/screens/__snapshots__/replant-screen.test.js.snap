// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Replant Screen renders correctly 1`] = `"<View style=\\"flex:1\\"><View style=\\"background-color:#472600;color:#FFFFFF;padding:20px;flex-direction:column;flex:1\\"><Text style=\\"color:#FFFFFF;font-size:32px;margin-bottom:20px;text-align:center\\">Start the seedlings</Text><Text style=\\"color:#FFFFFF;font-size:18px;margin-bottom:20px\\">Open the seed packets, and place two seeds in each hole of the rockwool, following the layout below. Once all necessary seeds have been placed in the rockwool, slide the seedling tray into the bottom layer of the greenhouse.</Text><RCTScrollView contentContainerStyle=\\"[object Object]\\" data=\\"\\" numColumns=\\"5\\" initialNumToRender=\\"10\\" maxToRenderPerBatch=\\"10\\" scrollEventThrottle=\\"50\\" updateCellsBatchingPeriod=\\"50\\" windowSize=\\"21\\" viewabilityConfigCallbackPairs=\\"\\" stickyHeaderIndices=\\"\\"><View><View><View style=\\"flex-direction:row\\"></View></View></View></RCTScrollView><View style=\\"background-color:#456E31;border-radius:5px;padding:10px;margin:10px;align-items:center;opacity:1\\"><Text style=\\"color:#FFFFFF;font-size:22px\\">Finish Replanting</Text></View></View></View>"`;
